modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -74,39 +74,32 @@ class NewProfileIndex extends React.Component {
    */
   handleSubmit(e) {
     e.preventDefault();
-    const match = this.validateMatchDate();
-    if (!match) {
-      this.setState({
-        errMessage: 'Date of Birth or EDC fields must match',
```

Comment:
you just removed the errors. where did they go?

Reply:
i see now that you're opting for the php errors instead of defining it in react which is fine! i'm wondering though if @ridz1208 cares about the validation to also be done on the front-end.

if not, you can remove the validateMatchData() method as it's not being used!

Reply:
I do, validation can definitely be done on the front end !!  But it must also exist on the backend in case the front end is... incapacitated